src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colour.values().length);
+        return Colour.values()[index].toString().toLowerCase();
```

Comment:
```suggestion
        return Colour.values()[index].name().toLowerCase();
```
[Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)